CleanArchitectureRxSwift/Scenes/CreatePost/CreatePostViewModel.swift

```diff
@@ -31,11 +31,7 @@ final class CreatePostViewModel: ViewModelType {
 
         let save = input.saveTrigger.withLatestFrom(titleAndDetails)
                 .map { (title, content) in
-                    return Post(uid: UUID().uuidString,
-                            createDate: Date(),
-                            updateDate: Date(),
-                            title: title,
-                            content: content)
+                    return Post(body: content, title: title, uid: 5, userId: 7)
```

Comment:
I'm wondering why did you change `uid` to be `Int` rather than `String`? I understand that it's probably because the API you're using have `Int` for the `id` field. But I wondering if it's possible to use `String` event though it's comping as `Int`. Then you won't need to do these hacks